app/src/org/commcare/google/services/analytics/FirebaseAnalyticsUtil.java

```diff
@@ -577,11 +577,11 @@ public static void reportNavDrawerItemSelected(String selectedItem) {
                 new String[]{selectedItem});
     }
 
-    public static void reportOtpRequested(int numberOfReattempts) {
+    public static void reportOtpRequested(int numberOfAttempts) {
         reportEvent(
                 CCAnalyticsEvent.OTP_REQUESTED_EVENT,
-                CCAnalyticsParam.OTP_REATTEMPTS,
-                String.format("%d reattempt(s)", numberOfReattempts)
+                CCAnalyticsParam.OTP_ATTEMPTS,
+                String.format("%d attempt(s)", numberOfAttempts)
```

Comment:
we should not really convert this into string, but directly use `bundle.putInt` to preserve the data type on this.